app/services/claims/submit.rb

```diff
@@ -1,18 +1,34 @@
 class Claims::Submit
   include ServicePattern
 
-  def initialize(claim:, claim_params:)
+  def initialize(claim:, claim_params:, user:)
     @claim = claim
     @claim_params = claim_params
+    @user = user
   end
 
   def call
     updated_claim.save!
+
+    # We will be implementing a different email for support users in another ticket
+    if user.type != "Claims::SupportUser" && status_changed_to_submitted
```

Comment:
You could use the `support_user?` method for this:

```suggestion
    if !user.support_user? && status_changed_to_submitted
```